src/directive.js

```diff
@@ -0,0 +1,83 @@
+import interpolate from './interpolate'
+import translate from './translate'
+import Vue from 'vue'
+
+// we have to cache the current language locally
+// to detect if the language has changed and translation should be updated
+let currentLanguage = null
```

Comment:
It's related to the spec: `updates a translation after a language change`

I couldn't find a way to know if the current language has change within the directive's `update` hook, so I needed a cached value to compare against `Vue.config.language` to determine if the translation should be updated. The code is around line 71